apps/src/p5lab/spritelab/commands/validationCommands.js

```diff
@@ -1,4 +1,11 @@
+import * as drawUtils from '@cdo/apps/p5lab/drawUtils';
```

Comment:
Can we add a comment at the top of this file explaining its overall purpose?

Reply:
The additional comments for each function are great, but it feels like this file is large and important enough to warrant a quick explanation at the top for somebody getting their bearings.